src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -1093,86 +1099,82 @@ export function getAlertContent(alert) {
+            <p>Board of Veterans’ Appeals P.O. Box 27063 Washington, DC 20038</p>
+          </div>
         ),
         displayType: 'take_action',
         type
       };
     }
     case ALERT_TYPES.rampEligible:
       return {
-        title: 'You have opted-in to the Rapid Appeals Modernization Program (RAMP)',
+        title: 'This appeal is eligible for the Rapid Appeals Modernization Program',
         description: (
           <div>
-            <p>You chose to participate in the new supplemental claim or
-            higher-level review lanes. This does not mean that your appeal has
-            been closed. If you didn’t choose this, please call your VSO or
-            representative as soon as possible.</p>
-            <p>At this time, Vets.gov Appeal Status is not able to get
-            information related to appeals that are part of RAMP.</p>
+            <p>On {formattedDate}, VA sent you a letter informing you of a new program called the Rapid Appeals Modernization Program (RAMP). The Veterans Appeals Improvement and Modernization Act will create new options in 2019 for Veterans seeking review of VA decisions. RAMP is a program that allows you to opt in to one of the two new "lanes" for review before the new law takes effect. For more information, review the fact sheet that was enclosed with the letter.</p>
+            <p>In order to participate, you must return the RAMP Opt-in Election form. If you choose to participate in RAMP, VA will withdraw all of your eligible appeals and instead review your case using the lane you select. If you do not want to participate in RAMP and wish to continue your appeal under the existing process, no action is required.</p>
           </div>
         ),
         displayType: 'info',
         type,
       };
     case ALERT_TYPES.rampIneligible:
       return {
-        title: 'Your appeal is not eligible for the Rapid Appeals Modernization Program (RAMP)',
+        title: 'This appeal is not eligible for the Rapid Appeals Modernization Program',
         description: (
-          <p>This appeal is not eligible for RAMP because it’s near the front
-          of the docket line and ready to be assigned to a Veterans Law Judge
-          at the Board. For more information, please call your VSO or
-          representative.</p>
+          <p>On {formattedDate}, VA sent you a letter informing you of a new program called the Rapid Appeals Modernization Program (RAMP). However, this appeal is not eligible to participate in RAMP because it {statusDescription}. If you have other appeals, they may still be eligible to participate in RAMP.</p>
         ),
         displayType: 'info',
         type,
       };
     case ALERT_TYPES.decisionSoon:
       return {
-        title: 'Your appeal is near the front of line',
+        // TODO: confirm which of duplicate content is correct
```

Comment:
@gnakm could you please confirm the content for decision_soon, it looks like there might be duplicate content provided for that alert type?

Reply:
https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Appeals%20Status/V2/Design%20files/Alerts.md#decision_soon

`decision_soon`
Alert title: Decision soon

Description: Your appeal will soon receive a Board decision. Sending in new evidence at this time could delay review of your appeal. If you’ve moved recently, please make sure that VA has your current mailing address.

Reply:
Thanks @gnakm, updated.